.github/workflows/main.yml

```diff
@@ -28,6 +28,8 @@ jobs:
         uses: SpicyPizza/create-envfile@v2.0.3
         with:
             envkey_SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+            MIXPANEL_PROJECT_TOKEN: ${{ secrets.MIXPANEL_PROJECT_TOKEN }}
```

Comment:
This GH action expect an `envkey_` prefix for each env var...
like we did for SENTRY_DSN

see docs for reference https://github.com/SpicyPizza/create-envfile